ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -23738,6 +23738,81 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```

Comment:
Anyone can point me how this prowjob hooks ci-operator/step-registry/cucushift/cucushift-workflow.yaml